gallery/src/styles/NxList/NxListBulletedExample.html

```diff
@@ -6,44 +6,42 @@
     distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
 
 -->
-<div class="nx-list nx-list--bulleted">
-  <h4 class="nx-list__title">
-    List title
-  </h4>
-  <ul>
-    <li class="nx-list__item">
-      item 1
-    </li>
-    <li class="nx-list__item">
-      <span>Item 2</span>
-      <ul class="nx-list nx-list--bulleted">
-        <li class="nx-list__item">
-          item 1
-        </li>
-        <li class="nx-list__item">
-          Item 2
-        </li>
-        <li class="nx-list__item">
-          <span>item 3</span>
-          <ul class="nx-list nx-list--bulleted">
-            <li class="nx-list__item">
-              item 1
-            </li>
-            <li class="nx-list__item">
-              Item 2
-            </li>
-            <li class="nx-list__item">
-              item 3
-            </li>
-          </ul>
-        </li>
-        <li class="nx-list__item">
-          Item 4
-        </li>
-      </ul>
-    </li>
-    <li class="nx-list__item">
-      Item 3
-    </li>
-  </ul>
-</div>
+<h3 class="nx-h3">
+  Bulleted lists
+</h3>
+<ul class="nx-list nx-list--bulleted">
+  <li class="nx-list__item">
+    <span class="nx-list__text">item 1</span>
```

Comment:
Minor: inconsistent capitalization among the list items

Reply:
> Minor: inconsistent capitalization among the list items

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/9c449dc1946410c44f70aa2f2ca8f18b337bae4e